src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -302,7 +305,11 @@ public Packaging.PackageBuilder CreateBuilder(string basePath, NuGetVersion vers
             // Add sources if this is a symbol package
             if (IncludeSymbols)
             {
-                ApplyAction(p => p.AddFiles(builder, SourcesItemType, SourcesFolder));
+                if (SymbolPackageFormat != SymbolPackageFormat.Snupkg)
```

Comment:
Why not do a positive check? 
A potential 3rd option would break this in the future. 
`SymbolPackageFormat == SymbolPackageFormat.SymbolsNupkg`